magfest/tests/test_templates.py

```diff
@@ -0,0 +1,8 @@
+import pytest
+
+from uber.tests import collect_template_paths, is_valid_jinja_template
```

Comment:
**NOTE**: This won't actually work until after https://github.com/magfest/ubersystem/pull/1888 is merged. But I verified that it works locally.